packages/create-remix/templates/arc/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "cross-env NODE_ENV=production arc sandbox"
```

Comment:
@mcansh: `NODE_ENV=development` is not required for any Architect processes, so in theory we can remove that from `start` + `dev:arc` if that doesn't cause any trouble for y'all!

Reply:
not 100% sure when we added that to the templates, but i'm 95% sure we set NODE_ENV during `remix dev` and `remix build` (and `remix-serve`)

Reply:
Ok, so do you want to leave it? Remove it? Arc is good either way, but imo always good to have less stuff if possible.

Reply:
let's remove it